### ISSUE TITLE
Review of administration functions

### ISSUE THREAD

**Issue by iangreenleaf** [grinnellplans/GrinnellPlans#9]
We need a realistic review of the [administration functions](https://github.com/annaswims/GrinnellPlans/wiki/Feature-List) and a list of those that are still needed.

**Comment by ghadirim** [comment]
Here are the current ones that are still used by me:
Add user
Change motd
Reset password
Delete user
Moderate secrets


I will think about others that are still needed. Nothing pops into my head besides having a master list of who is blocked by whom to be able to see if anyone in particular is causing problems for a larger group of people, but I think admins should probably discuss that a bit.

**Comment by iangreenleaf** [comment]
Thanks!

"Email" is really "Change email", which I'm guessing is useful but we'll probably just roll into one function that allows editing a user to change the email and/or password.

So the ones that are set to get cut are:

* Set per-account messages
* Administer donations list
* Create a poll (maybe, I feel like this is something that is not useless, but probably not required for launch)
* View stylesheet usage stats
* Switch a user's password with that of [test].
* Data dump of update stats
* Data dump of new plan stats

@ghadirim, check with the other admins if you need to and let us know if we should be keeping any of those. Otherwise we'll proceed with this plan. Creating tickets now to cover the needed admin stuff.

**Comment by iangreenleaf** [comment]
Actually, I just thought about changing email and password some more. Do you need an admin interface to reset passwords, or can you just direct people to the "Forgot password" page? I can still see how you might occasionally need to change an email, though. Like if somebody has a super old email (like their Grinnell address) and now can't log in and can't receive the password reset email. Does that make sense?

**Comment by iangreenleaf** [comment]
Think we covered all the important ones. Email and password are both changeable.
